lib/elixir/test/elixir/task_test.exs

```diff
@@ -495,6 +495,16 @@ defmodule TaskTest do
              {:timeout, {Task.Supervised, :stream, [0]}}
       refute_received _
     end
+
+    test "streams an enumerable with ordered: false" do
```

Comment:
We also need a test for the `exit: reason`. Also we should probably make it `{:exit, position, reason}`.